_shared/project/conf/supervisord.conf

```diff
@@ -0,0 +1,34 @@
+[supervisord]
```

Comment:
Same deal as with `supervisord-dev.conf`: this is the same file that's already in the `pyramid-app` cookiecutter on `main` just moved into `_shared` and with the `[program:app]` section for `pyapp`'s added. As with `supervisord-dev.conf` we do want to have _something_ in `supervisord.conf` by default so that the `Dockerfile` has something to run by default (for example: there is something to happen when you run `make docker-run`).